### PR TITLE
Fetch FiFi test results from IQE for our tooling to use

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: tests/foreman/api/test_fifi.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@pytest.mark.tier2
+def test_fifi():
+    """Fetch FiFi test result from IQE's Jenkins
+
+    :id: d63f87e5-66e6-488a-8c44-4129259494a6
+
+    :expectedresults: There are some tests found and all of the passed
+    """
+    job = requests.get(
+        'https://insights-stg-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/'
```
Comment (jameerpathan111) [inline tests/foreman/api/test_fifi.py:37]: @lhellebr we should add this urls to robottelo.properties files.

**Review by jameerpathan111 — CHANGES_REQUESTED** [review]

File: tests/foreman/api/test_fifi.py (hunk 1 above)
Comment (jameerpathan111) [reply tests/foreman/api/test_fifi.py:37]: our automation will replace it later with right url.

Comment (lhellebr) [reply tests/foreman/api/test_fifi.py:37]: Of course, I planned that and forgot. Will do :)

Comment (lhellebr) [reply tests/foreman/api/test_fifi.py]: Fixed in this PR. TODO: set this up in automation config.

File: tests/foreman/api/test_fifi.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    ).json()
+    last_build_number = job['lastCompletedBuild']['number']
+    all_tests = requests.get(
+        f'{jenkins}/{fifi_job}/{last_build_number}/testReport/api/json'
+    ).json()['suites'][0]['cases']
+    tests = [
+        test for test in all_tests if 'sat69' in test['name']
+    ]  # select those with matching Satellite version
+    test_count = len(tests)
+    successful_test_count = sum(1 for test in tests if test['status'] == 'PASSED')
```
Comment (jameerpathan111) [inline tests/foreman/api/test_fifi.py:52]: This means even if single test failed on iqe side we will be marking robottelo one as failed, right?

Comment (jameerpathan111) [reply tests/foreman/api/test_fifi.py:52]: pre-commit check is failing. :|

Comment (lhellebr) [reply tests/foreman/api/test_fifi.py:52]: Right. After this is marked as failed, everyone can take a look into IQE jenkins for more granularity.

**Review by jameerpathan111 — APPROVED** [review]
LGTM

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
We need to discuss the intent and desirability of this addition.

**Comment by lhellebr** [comment]
This is to make FiFi status visible in our tooling, such as Polarion or Report Portal.

**Comment by JacobCallahan** [comment]
Closing this as we will be taking a different approach to gather these results.

**Pull request closed without merging**